sadu-mapper/src/main/java/de/chojo/sadu/wrapper/util/Row.java

```diff
@@ -339,7 +339,11 @@ public Time getTime(int columnIndex) throws SQLException {
      *                      called on a closed result set
      */
     public LocalTime getLocalTime(int columnIndex) throws SQLException {
-        return getTime(columnIndex).toLocalTime();
+        var time = getTime(columnIndex);
+        if (time != null) {
+            return time.toLocalTime();
+        }
+        return null;
```

Comment:
this can be easily be replaced by a ternary operator.
```suggestion
        return time == null ? null : time.toLocalTime();
```